woocommerce/woocommerce-android
# Issue/5711 interac refund error handling

part of: #5711  


### Description

This PR handles error case scenarios for Interac refunds. Most of the Interac refund failures will be handled with a generic "refund failed" screen. The reason for that is being the Stripe terminal SDK doesn't properly communicate the error code to the app. You can find more about this in the conversation [here](https://github.com/woocommerce/woocommerce-android/pull/p1647343856332389-slack-C01G168NFC2)

### Testing instructions

Green CI should be enough. In order to see what the end product looks like, please refer to [this](https://github.com/woocommerce/woocommerce-android/pull/6157) PR.

### Images/gif
